### ISSUE TITLE
Build: English content on French page

### ISSUE THREAD

**Issue by qanitah** [mmiron-HC/WST-RSAMS#67]
URL: https://hcdev.openplus.ca/fr/node/2525/build#Section1 (click on Build for any Recall or Alert from Dashboard)
User Agent: Chrome, Firefox
Language: French

Issue Description: The term Build is used in English and Générer is used in French. But for the h1, it's still Build. Tableau de bord also uses the term Générer. It's also incorrect for page title. 

Steps to reproduce: Check h1 and page title on the French page.

Expected results: The text should be in French.

![image](https://user-images.githubusercontent.com/14280062/129095356-7a50cce3-92f2-4a1d-924a-b600eebbbaa8.png)

**Comment by qanitah** [comment]
Fixed
